lib/Guideline/index.js

```diff
@@ -26,9 +28,13 @@ class Guideline extends Component {
   }
 
   toggleShow = () => {
-    this.setState(prevState => ({
-      showContent: !prevState.showContent
-    }));
+    this.setState(
+      prevState => ({
+        showContent: !prevState.showContent
+      }),
+      () => this.props.onToggle(this.state.showContent)
```

Comment:
💚As `this.state.showContent` is set asynchronously, I don't think we can guarantee that `this.state.showContent` is the state that has just been set.

I'm pretty sure this is edge case, as someone would have to be clicking the toggle pretty fast for it to get out of whack. Does the `setState` callback pass the updates state as its first arg?